WLogger/WeekLogs/WeekLogsTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  WeekLogsTableViewController.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class WeekLogsTableViewController: UITableViewController {
+
+    var viewModel: WeekLogsTableViewModel?
+}
+
+extension WeekLogsTableViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.refreshControl = UIRefreshControl()
+        self.tableView.addSubview(refreshControl!)
+        self.refreshControl?.addTarget(self, action: "handleRefresh:", forControlEvents: UIControlEvents.ValueChanged)
+        tableView.registerNib(UINib(nibName: "LogTableViewCell", bundle: nil), forCellReuseIdentifier: LogTableViewCellIdentifier)
```

Comment:
I am not saying to this right now but in Syrmo we try to avoid string as identifier by all means. Here is what we do

https://github.com/syrmo/syrmo-ios/blob/master/Syrmo/Identifiers.swift#L11

and we extend classes that need identifier like

https://github.com/syrmo/syrmo-ios/blob/73907c8693872c0a9cd228ffedd295fa132dbf4d/Syrmo/UIExtensions/NSBundleExtension.swift